pom.xml

```diff
@@ -439,6 +439,11 @@
             <name>Conveyal Maven Repository</name>
```

Comment:
Are there other artifacts being pulled from the Conveyal Maven Repo, or was the forked oba-gtfs modules the only one?  If more, this is fine as-is.  If no others, then this repo should be removed.

Reply:
Yeah, there are a few OSM libraries that come from the Conveyal repo.

Reply:
I would prefer to eliminate this Conveyal repo and use only Conveyal libraries that are released via Maven Central, if any at all.

One other library coming from Conveyal is `jackson2-geojson` - we should really try to move away from using our own custom fork of Jackson. I believe GeoJSON functionality may have been added to Jackson or some other common JSON libraries recently - we should research that.

Is the other library crosby.binary.osmpbf? I guess OTP still contains its own code for reading OSM data. It escapes me now why we would have our own fork of this. Conveyal has since created a library for that (osm-lib) that should probably be used in a future version of OTP. Conveyal osm-lib does not have this dependency on crosby.binary.osmpbf, instead relying on the (more recent?) org.openstreetmap.osmosis.osmosis-osm-binary.

I'll create another ticket for this - reducing dependencies on forked libraries.

Reply:
Yep, just osmpbf.

In general, if there's a maven hosting concern, we have a Nexus repository for OBA we can offer for hosting. (Though I understand the main concern is having dependencies that are stable/vanilla/known.)

Reply:
Thanks for the offer @sdjacobs. In fact while it's very basic, the Conveyal system using the "S3 wagon" to host a Maven repo is simple to maintain and works well enough. As you said, the main concern is not the nature of that repo, but the fact that we've forked away from stable, well-maintained projects.